src/openforms/appointments/contrib/jcc/tests/test_plugin.py

```diff
@@ -0,0 +1,144 @@
+import os
+from datetime import date, datetime
+
+from django.test import TestCase
+
+import requests_mock
+from zeep.client import Client
+
+from openforms.appointments.contrib.base import (
+    AppointmentClient,
+    AppointmentLocation,
+    AppointmentProduct,
+)
+
+from ..plugin import Plugin
+
+
+def mock_response(filename):
+    filepath = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), "mock", filename)
+    )
+    with open(filepath, "r") as f:
+        return f.read()
+
+
+class PluginTests(TestCase):
+    maxDiff = 1024
+
+    def setUp(self, *args, **kwargs):
+        self.plugin = Plugin()
+
+        wsdl = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), "mock/GenericGuidanceSystem2.wsdl")
+        )
+        self.plugin.client = Client(wsdl)
+
+    def test_get_available_products(self):
+        with requests_mock.mock() as m:
+            m.post(
+                "http://example.com/soap11",
+                text=mock_response("getGovAvailableProductsResponse.xml"),
+            )
+
+            products = self.plugin.get_available_products()
+            self.assertEqual(len(products), 3)
+
+            self.assertEqual(products[0].identifier, "1")
+            self.assertEqual(products[0].code, "PASAAN")
+            self.assertEqual(products[0].name, "Paspoort aanvraag")
```

Comment:
```suggestion
            products = self.plugin.get_available_products()
            
            self.assertEqual(len(products), 3)
            self.assertEqual(products[0].identifier, "1")
            self.assertEqual(products[0].code, "PASAAN")
            self.assertEqual(products[0].name, "Paspoort aanvraag")
```
The block structure of

```
setup1
setup2

call_thing_to_test

assert1
assert2
```

makes debugging/understanding tests easier on the eyes :)